src/그리디/01-거스름돈.js

```diff
@@ -0,0 +1,32 @@
+
+// n : 거스름돈
+
+// 내가 푼 방법
+
+function solution(n){
+    let count = 0;
+    const units = [500,100,50,10];
+    let index = 0;
+    while(n > 0 && index < 4){
+        if(units[index] > n){
+            index++;
+        }
+        n-=units[index];
+        count++;
+    }
+    return count;
+}
+
+// 책에 나와있는 답안
+
+function solution2(n){
+    let count = 0;
+    [500,100,50,10].forEach((coin)=>{
+        count += (Math.floor(n/coin));
+        n %= coin;
+    });
+    return count;
+
+}
+
+console.log(solution2(1260));
```

Comment:
```suggestion
console.log(solution2(1260));

```



`No newline at end of file` 이라고 깃헙에서 얘기하고 있네요! 
![image](https://user-images.githubusercontent.com/31029000/134900566-17ee32ba-2d5a-44fe-b6cb-620239baf84e.png)

Reply:
앗 넵 감사합니다 ㅎㅎ